custom/icds_reports/ucr/reports/ls/ls_v2_report_child_names.json

```diff
@@ -135,7 +135,10 @@
         "aggregation": "simple",
         "column_id": "dob",
         "field": "dob",
-        "transform": {},
+         "transform": {
+      "type": "date_format",
```

Comment:
@SunainaWalia can you fix the indentation in these files?

Reply:
fixed the indentation @mkangia